_pytest/python.py

```diff
@@ -131,6 +131,7 @@ def pytest_namespace():
     raises.Exception = pytest.fail.Exception
     return {
         'raises': raises,
+        'not_raises': NotRaisesContext,
```

Comment:
> oh, an additonal note, why do we make it a callable?

It felt more natural, because you can use `pytest.raises(...)`. No problem changing it to a singleton if you feel it is more appropriate.

> else we might see people wanting to creep in more specific behavior causing grief down the line

What do you mean? People asking to some weird parameter to it to give additional behavior? Can't we just say *no*?